backend/tests/FuzzTests/ExecutePureFunctions.FuzzTests.fs

```diff
@@ -145,7 +146,8 @@ module Generators =
           // (to test edges of conversions to strings, and just be more robust in general).
           // If we're able to remove `containsBytes` or reduce its scope, all the better.
           let! bytes = Arb.generate<byte []>
-          let v = RT.EString(gid (), Base64.defaultEncodeToString bytes)
+          let v =
+            RT.EString(gid (), [ RT.StringText(Base64.defaultEncodeToString bytes) ]) //CLEANUP
```

Comment:
There's a CLEANUP here and in a few other places, not sure what the plan was there?

Reply:
After implementing string interpolation, we fixed the errors in EString's string parameters by wrapping them in StringText(). Stachu suggested marking them as CLEANUP, and then see whether this approach is the enough to solve the issue or if we need a different approach.